vagrant/provision.sh

```diff
@@ -18,5 +19,8 @@ HOME_DIR="${HOME_DIR:-/home/vagrant}"
 VAGRANT_USER="${VAGRANT_USER:-vagrant}"
 
 HOME_DIR=$HOME_DIR SWAN_USER=$VAGRANT_USER $HOME_DIR/go/src/github.com/intelsdi-x/swan/vagrant/provision_experiment_environment.sh
-HOME_DIR=$HOME_DIR SWAN_USER=$VAGRANT_USER $HOME_DIR/go/src/github.com/intelsdi-x/swan/vagrant/provision_development_environment.sh
-HOME_DIR=$HOME_DIR SWAN_USER=$VAGRANT_USER $HOME_DIR/go/src/github.com/intelsdi-x/swan/vagrant/provision_ci_environment.sh
+
+if [ -v SWAN_DEVELOPEMENT_ENVIRONMENT ]; then
```

Comment:
@skonefal - I guess that we need to modify Jenkins configuration. Have you talked to @st4nson or @ppadamcz?